pra_request_tracker/record_requests/tests/test_models.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from .factories import AgencyFactory
+from .factories import RecordRequestFactory
+from .factories import RecordRequestFileFactory
+
+from ..models import Agency, RecordRequestFile
+from ..models import RecordRequest
+
+
+@pytest.mark.django_db(transaction=True)
```

Comment:
A couple of options here (based off [the documentation](https://pytest-django.readthedocs.io/en/latest/helpers.html#pytest-mark-django-db-request-database-access)):
1. We could use `transactional_db` as a function parameter
2. We could set this decorator to a variable and then @ it for every entry (e.g. `transactional_db = pytest.mark.django_db(transaction=True)` and then `@transactional_db` above the test)

Thoughts? I just hate to have to specify this every time and I feel like I'd forget in some cases

Reply:
Sure! FWIW the test cases will yell at you if you forget to add it and touch the DB at all.

I think ideally we'd enable this for all tests? Or just use the Django TestCase class which I think includes all of this for free.

Reply:
Oh good to know! I've used `django_db` as a marker before but I haven't typically set `transaction=True`. I must say I like function based tests over the class based ones, so the aliasing seems solid!

Reply:
I converted them to use the Django TestCase. I think that's a better general approach than using pytest.

Reply:
Oop, you already converted! All good :+1: